### PR TITLE
Refresh /flavours page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Note: Will need a a new banner image as the current one is too low resolution** 
- Updates /flavours ready for 22.04 refresh, as per [copydoc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit#)

## QA

- go to https://ubuntu-com-11489.demos.haus//desktop/flavours, compare against [copydoc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5109

### CODE REVIEW

**Comment by pmahnke** [comment]
I think the CTAs are in the wrong order... should be:

[ GREEN BUTTON ]
text cta 1 >
text cta 2 >

or even 

[ GREEN BUTTON ]
text cta 1 >   text cta 2 >

**Comment by pmahnke** [comment]
also, why not use `.p-strip--suru-background` for the banner with the image? and are you getting a new image?

**Comment by local-optimum** [comment]
Feedback:
- Banner image needs redoing
- It would be good to have the screenshots a lot bigger
- Missing the intro text at the top of the copy doc
- I need to add some 'other things to do' at the bottom as it's a bit empty at the mo, will add to the copy

File: templates/desktop/flavours.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+              height="64",
+              width="64",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img"}
+            ) | safe
+          }}
+          <h3 class="p-heading-icon__title">Kubuntu</h3>
+        </div>
+        <p>Kubuntu unites Ubuntu with KDE and the fabulous Plasma desktop, bringing you a full set of applications including productivity, office, email, graphics, photography, and music applications ready to use at startup with extensive additional software installed from not one, but two desktop package managers. </p>
```
Comment (anthonydillon) [inline templates/desktop/flavours.html:36]:
```suggestion
        <p>Kubuntu unites Ubuntu with KDE and the Plasma desktop, bringing you a full set of applications including productivity, office, email, graphics, photography, and music applications ready to use at startup with extensive additional software installed from not one, but two desktop package managers. </p>
```

File: templates/desktop/flavours.html [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img"}
+            ) | safe
+          }}
+          <h3 class="p-heading-icon__title">Ubuntu Budgie</h3>
+        </div>
+        <div>
+          <p>Ubuntu Budgie is a proud official Ubuntu flavour. We combine the simplicity and elegance of the Budgie desktop environment with the power and familiarity of an Ubuntu based operative system.</p>
+          <p>The result is a modern and fast desktop distribution with great defaults, yet fully customizable.</p>
+          <p><a href="https://lubuntu.me" class="p-button--positive">Get Ubuntu Budgie</a></p>
```
Comment (anthonydillon) [inline templates/desktop/flavours.html:117]:
```suggestion
          <p><a href="https://ubuntubudgie.org/downloads/" class="p-button--positive">Get Ubuntu Budgie</a></p>
```

**Review by anthonydillon — APPROVED** [review]
LGTM with 2 small changes

**Pull request merged**